src/main/java/io/codelex/test/exercise2/Basket.java

```diff
@@ -0,0 +1,30 @@
+package io.codelex.test.exercise2;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Basket<N, T> {
+    private final Map<N, T> basket = new HashMap<>();
```

Comment:
Interesanti, kāpēc Map, nevis vienkārši List? :) Šo vajadzētu pārtaisīt par List un tad arī bīs vienkāršāks kods un tikai viens Generic tips.

Reply:
Jā, es samulsu pie  - Basket can store only one kind of items. Un tad domāju, ka Map var palīdzēt ar unikālo tipu glabāšanu, bet uz beigām jau likās, ka tomēr nepareizi, jo List<T> jau norāda unikālo tipu. Paldies, izlaboju!